<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StokBarang</class>
 <widget class="QMainWindow" name="StokBarang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StokBarang</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuDelete">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add...</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
